client/source/js/modules/ui/modal/modal-service.js

```diff
@@ -21,15 +21,24 @@ define([
          * executes the right callback depending on the user's choice.
          */
         confirm: function (onAccepted, onRejected, acceptButton, rejectButton, message, title) {
-          $modal.open({
+
+          var onModalKeyDown = function (anEvent) {        
```

Comment:
And I forgot to mention that, that naming together with good method naming, makes it closer to spoken english:
`this.doSomethingWith(anObject)`
`this.convertUsing(aConverter)`
`moneyController.includes(somePayments)`
`carModel.removeTires(someTires)`
etc